src/index.css

```diff
@@ -69,3 +70,68 @@ button:focus-visible {
     background-color: #F9F9F9;
   }
 }
+
+h2 {
+  font-size: 36px;
+  font-weight: 700;
+  color: #FFF;
+}
+
+@media screen and (min-width: 390px) and (max-width: 736px) {
+
+  h2 {
+    font-size: 19px;
```

Comment:
Please correct font-size, line-height and font-weight according to the design

Reply:
trying to make